nifi-nar-bundles/nifi-azure-bundle/nifi-azure-reporting-task/src/main/java/org/apache/nifi/reporting/azure/loganalytics/AbstractAzureLogAnalyticsReportingTask.java

```diff
@@ -136,7 +136,7 @@ protected void sendToLogAnalytics(final HttpPost request, final String workspace
 
         final int bodyLength = rawJson.getBytes(UTF8).length;
         final ZonedDateTime zNow = ZonedDateTime.now(ZoneOffset.UTC);
-        final String nowRfc1123 = zNow.format(DateTimeFormatter.RFC_1123_DATE_TIME);
+        final String nowRfc1123 = zNow.format(DateTimeFormatter.ofPattern("EEE, dd MMM yyyy HH:mm:ss O"));
```

Comment:
Based on my reading of RFC-1123 and the [Azure Log Analytics docs](https://docs.microsoft.com/en-us/rest/api/loganalytics/create-request), I think it would be better to use `GMT` in place of `0` in the tz/offset field
```suggestion
        final String nowRfc1123 = zNow.format(DateTimeFormatter.ofPattern("EEE, dd MMM yyyy HH:mm:ss GMT"));
```

Reply:
Nevermind, I read the DateFormatter docs too quickly... this seems correct to me. Will do a quick test and should be able to approve this